build-plugin/action.yml

```
@@ -88,6 +92,12 @@ runs:
       run: if [ "v${{ steps.package-plugin.outputs.plugin-version }}" != "${{ steps.metadata.outputs.github-tag }}" ]; then printf "\033[0;31mPlugin version doesn't match tag name. The tag should be v${{ steps.package-plugin.outputs.plugin-version }} \033[0m\n"; exit 1; fi
       shell: bash
 
+    - name: Generate artifact attestation
+      if: ${{ inputs.attestation == 'true' }}
```

Similar to above, can we make `inputs.attestation` of `type: boolean` so this becomes:

```suggestion
      if: ${{ inputs.attestation }}
```